docs/_posts/2019-03-05-go.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: "Getting GOing"
+date: 2019-03-05 09:00:00 -0600
+author: Sigal Sax
+categories: blog
+published: true
+image: ship_it.jpg
+thumb: ship_it.jpg
+image-alt: Test post
+excerpt: "For the past year, we on the Cyberark Conjur team have been working on our latest open source project in Go..."
+---
+
+From the very beginning, the original Golang developers had a clear goal - retain the positive attributes of the common programming languages while building one that was readable, simple, and high performing. Go’s light-weight model for fast compilation, extensive toolset, and native capabilities are what have propelled it toward worldwide adoption and success.  With these modern features and being backed by Google, it is no wonder why open source projects such as Docker, Kubernetes, and some in Cyberark, are being built in Go.  
+
+## Get GOing with Secretless  
+
+For the past year, we on the Cyberark Conjur team have been working on our latest open source project in Go, the Secretless Broker. The Secretless Broker acts as a transparent proxy that injects the necessary credentials to target services directly, relieving client applications of the responsibility of handling sensitive data. Essentially, you can open a local connection to a credentialed resource without ever having to pass credentials to do so; all with Secretless. By removing the direct interaction between applications and target resources, the potential for a secret leak decreases significantly. 
+
+I was new to this project and especially new to the Go community, so naturally, I was curious why we made the design decision to build the Secretless Broker in Go. After working on features, participating in design discussions and debates during team standups, and having just attended the Go conference, it became clear why. We needed a system-level language that was as fast as the C-family but not as low level. Go gave us the perks of a low-level language while providing us with a toolset and native capabilities of a higher-level one.  
```

Comment:
instead of "the Go conference" can  you say "the inaugural GopherCon Israel" and link to the conf?

Reply:
see below changes